Sources/Build/BuildPlan.swift

```diff
@@ -795,7 +807,7 @@ public final class SwiftTargetBuildDescription {
         result.append("-I")
         result.append(buildParameters.buildPath.pathString)
 
-        result += self.compileArguments()
+        result += try self.compileArguments()
```

Comment:
As noted, I'm not sure that constructing the compilation arguments should be a throwing function since the only error is to be unable to determine whether we can rename the entry point, which I think should be treated the same as not being able to.

Reply:
Thanks for this change!  To summarize the comments, I think it might be better to:
- consolidate the check for the specific flag into a separate property (even if it's implemented in terms of the more general function) so that not every use has to repeat the name of the flag
- not have the function be a throwing function, but to instead return false if error
- take the "Xfrontend" out of the check since it doesn't serve any purpose to check for that flag